editor/icons/FadeCross.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Needs to be optimized with SVG cleaner. (same with others)

Reply:
okay done